__tests__/services/users.service.spec.ts

```diff
@@ -51,13 +60,16 @@ describe('compare password', () => {
 
   it('checks that the password matches', () => {
     const hashedPassword = 'password';
-    const result = usersService.comparePassword(hashedPassword, userPassword);
+    const result = usersService.comparePassword({
+      password: hashedPassword,
+      userPassword
+    });
     expect(result).toBeTruthy();
   });
 
-  it('checks that the password don\'t match', () => {
+  it('checks that the password dont match', () => {
```

Comment:
Actually, isn't the other way better 🤔 ? 
@Alxzu wdyt?

Reply:
I honestly do not know which way is better so I opted for the most basic.
If we can get some extra point of view it would be good!

Reply:
Agree @agustinpurita , I don't have a strong opinion here

Reply:
I don't have either a strong opinion about this... but it looks clearer

Reply:
what is the output of the test in the console when it fails?

Reply:
the ouput when the fails looks good:` checks that the password don't match`

Reply:
nicee 😎

Reply:
thanks to both for the review!